### PR TITLE
chore(jasmine): remove jasmine 1.3

### PR DESCRIPTION
**Pull request by hankduan**
and update docs. Also, use jasmine 2 for running all Protractor's unit tests.

BREAKING CHANGE:
Now, both jasmine and jasmine2 frameworks use jasmine 2.3. Users still using
jasmine version <2 will have to upgrade.

### CODE REVIEW

File: debugging/failureConf.js [hunk 1]
```diff
@@ -17,9 +17,6 @@ exports.config = {
 
   // ----- Options to be passed to minijasminenode.
```
Comment (juliemr) [inline debugging/failureConf.js]: Change this text to 'Options to be passed to jasmine'

File: debugging/timeoutConf.js [hunk 2]
```diff
@@ -17,9 +17,6 @@ exports.config = {
 
   // ----- Options to be passed to minijasminenode.
```
Comment (juliemr) [inline debugging/timeoutConf.js]: Ditto.

File: docs/faq.md [hunk 3]
```diff
@@ -106,17 +106,6 @@ The method to take a screenshot automatically on failure would depend on the typ
...
-  this.reportSpecResults = function(spec) {
-    if (!spec.results().passed()) {
-      //take screenshot
-    }
-  };
-});
-```
-
-```javascript
 // Note: this is using Jasmine 2.1 reporter syntax.
```
Comment (juliemr) [inline docs/faq.md]: Let's change this comment to 'using Jasmine 2 reporter syntax'

File: website/test/e2e/api_spec.js [hunk 4]
```diff
@@ -114,7 +114,7 @@ describe('Api', function() {
 
     // Then ensure the child functions are shown.
     expect(apiPage.getChildFunctionNames()).toEqual([
-      'getProcessedConfig', 'forkNewDriverInstance', 
+      'getProcessedConfig', 'forkNewDriverInstance', 'useAllAngular2AppRoots',
```
Comment (juliemr) [inline website/test/e2e/api_spec.js]: Thanks, my bad for missing this one.

**Comment by hankduan** [comment]
merged with https://github.com/angular/protractor/commit/2bde92b3e745e09ad3876932b2d187365e9aaa31

**Pull request closed without merging**